applications/vanilla/controllers/api/DiscussionsApiController.php

```diff
@@ -567,6 +567,8 @@ public function patch($id, array $body) {
         $discussionData = ApiUtils::convertInputKeys($body);
         $discussionData['DiscussionID'] = $id;
         $categoryID = $row['CategoryID'];
+        $category = $this->categoryModel->getID($categoryID);
```

Comment:
We can use CategoryModel::categories($categoryID),
https://github.com/vanilla/vanilla/blob/ad5b3d07138fd32f0820d02cb7d0eaa983f8035e/applications/vanilla/models/class.categorymodel.php#L518
It uses cache around here 

https://github.com/vanilla/vanilla/blob/995750f30a8c77c4d01e85639b08aa4ceaed4971/applications/vanilla/library/class.categorycollection.php#L177